test/common_utils.py

```diff
@@ -492,6 +492,34 @@ def get_num_channels(color_space):
     return num_channels
 
 
+def make_image(
+    spatial_size,
+    *,
+    color_space="RGB",
+    batch_dims=(),
+    dtype=torch.float32,
+    device="cpu",
+    constant_alpha=True,
+    memory_format=torch.contiguous_format,
+):
+    spatial_size = _parse_spatial_size(spatial_size)
```

Comment:
We can be more strict here if we like. Currently also supports passing a single integer to be treated as edge size of a square image. I'm ok with only taking two-tuples (or sequences in general). LMK what you prefer.

Reply:
Only introduce it when/if we need it?